frontend/src/components/App.js

```diff
@@ -62,5 +62,7 @@ function mapDispatchToProps (dispatch) {
 
 export default connect(
   mapStateToProps,
-  mapDispatchToProps
-)(App)
\ No newline at end of file
+  mapDispatchToProps,
+  null,
+  { pure: false }
```

Comment:
pure: false-- https://github.com/reactjs/react-redux/blob/master/docs/troubleshooting.md#my-views-arent-updating-when-something-changes-outside-of-redux